l10n_it_fatturapa_in_rc/models/account_invoice.py

```diff
@@ -1,23 +1,22 @@
+#  Copyright 2023 Simone Rubino - Aion Tech
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
 from odoo import _, api, models
 from odoo.tools import float_compare
 
 
 class InvoiceLine(models.Model):
     _inherit = "account.move.line"
 
-    @api.depends(
-        "move_id",
-        "move_id.move_type",
-        "move_id.fiscal_position_id",
-        "move_id.fiscal_position_id.rc_type_id",
-        "tax_ids",
-    )
+    @api.depends()
     def _compute_rc_flag(self):
         if "fatturapa.attachment.in" in self.env.context.get("active_model", []):
             # this means we are importing an e-invoice,
             # so RC flag is already set, where needed
-            return
-        super()._compute_rc_flag()
+            for line in self:
+                line.rc = line.rc
```

Comment:
come mai questo?

Reply:
Perché i metodi dei campi calcolati devono sempre assegnare i campi che stanno calcolando:
> Fields can be computed (instead of read straight from the database) using the `compute` parameter. **It must assign the computed value to the field.**

(da https://www.odoo.com/documentation/16.0/developer/reference/backend/orm.html#computed-fields)